tests/label.js

```diff
@@ -0,0 +1,15 @@
+const promisifyAll = require('promise-toolbox').promisifyAll;
+var fatfs = require("../");
+var fatfsBuffer = require('./fatfs-buffer').buffer
+var fatfsBufferInit = require('./fatfs-buffer').init
+var fs = require('fs')
+
+const buffer = fatfsBufferInit({ label: 'cidata     ' })
+
+const { createLabel } = promisifyAll(
+  fatfs.createFileSystem(fatfsBuffer(buffer), {allowLowercaseNames: true})
+)
+
+createLabel("cidata").then(() => {
```

Comment:
I don't think it's worth bringing in a new dependency just for one call. Please match the "old school" callback style used by the rest of the project here.

Reply:
Fixed